server/accounts.go

```diff
@@ -3233,6 +3233,7 @@ func (s *Server) updateAccountClaimsWithRefresh(a *Account, ac *jwt.AccountClaim
 	for sub, wm := range ac.Mappings {
 		mappings := make([]*MapDest, len(wm))
 		for i, m := range wm {
+			m := m
```

Comment:
Why the heck is this needed?

Reply:
to make `m` a different instance per iteration, in Go 1.22 that will be default behavior but in Go 1.21 can already try to find the for loops that will be auto fixed to test in advance: https://github.com/golang/go/wiki/LoopvarExperiment#if-the-proposal-is-accepted-how-will-the-change-be-deployed || https://go.googlesource.com/proposal/+/master/design/60078-loopvar.md#a-decade-of-experience-shows-the-cost-of-the-current-semantics

Reply:
ok I will read up on it later but the fix, m := m looks kinda silly (not you just in general).. So rather find a better solution than that..

Reply:
I think you read it wrong, they are trying to fix the closure capture problem..